devicetypes/qubino/qubino-flush-dimmer-0-10V.src/qubino-flush-dimmer-0-10V.groovy

```diff
@@ -0,0 +1,671 @@
+
+    if (childDevice != null) {
+        sendHubCommand(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 1, scale: 0).format())
+    }
+
+    // refresh a child illuminance sensor (if available)
+    childDni = "${device.deviceNetworkId}:" + childDeviceSpecifics["illuminance"].childDni
+    childDevice = childDevices.find { it.deviceNetworkId == childDni }
+
+    if (childDevice != null) {
+        sendHubCommand(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 3, scale: 1).format())
+    }
+}
+
+private getParameterMap() {[
+	[
+		name: "Input 1 switch type", key: "input1SwitchType", type: "enum",
+		parameterNumber: 1, size: 1, defaultValue: 0,
+		values: [
+			0: "Default value - Mono-stable switch type (push button) – button quick press turns between previous set dimmer value and zero)",
+			1: "Bi-stable switch type ",
+			2: "Potentiometer (Flush Dimmer 0-10V  is using set value the last received from potentiometer or from z-wave controller)",
+			3: "0-10V Temperature sensor (regulated output)",
+			4: "0-10V Illumination sensor (regulated output)",
+			5: "0-10V General purpose sensor (regulated output)"
+		],
+		description: "Set input based on device type (switch, potentiometer, 0-10V sensor,..)." +
+			"After parameter change to value 3, 4 or 5 first exclude module (without setting parameters to default value) then wait at least 30s and then re include the module! "
+	],
+	[
+		name: "Auto or manual selection", key: "autoOrManualSelection", type: "boolean",
+		parameterNumber: 52, size: 1, defaultValue: 0,
+		optionInactive: 0, inactiveDescription: "Default value - Manual",
+		optionActive: 1, activeDescription: "Auto",
+		description: "This parameter is influencing on the software only when the value of parameter number 'Input 1 switch type' is set to value 3, 4 or 5. " +
+			"In manual mode regulation (how the input influence on output) is disabled. "
+	],
+	[
+		name: "Input I1 Sensor reporting", key: "inputI1SensorReporting", type: "range",
+		parameterNumber: 140, size: 2, defaultValue: 5,
+		range: "1..10000",
+		description: "Input I1 Sensor reporting If analogue sensor is connected, module reports measured value on change defined by this parameter. " +
+			"0 Reporting disabled, " +
+			"5 (Default value) = 0,5 change, " +
+			"1 - 10000 = 0,1 - 1000 step is 0,1"
+	],
+	[
+		name: "Enable/Disable Double click function", key: "enable/DisableDoubleClickFunction", type: "boolean",
+		parameterNumber: 21, size: 1, defaultValue: 0,
+		optionInactive: 0, inactiveDescription: "Default value - Double click disabled",
+		optionActive: 1, activeDescription: "Double click enabled",
+		description: "If Double click function is enabled, a fast double click on the push button will set dimming power at maximum dimming value. " +
+			"Valid only if input is set as mono-stable (push button)."
+	],
+	[
+		name: "Unsecure / Secure Inclusion", key: "unsecure/SecureInclusion", type: "boolean",
```

Comment:
not sure how this would even be used, since you'd have to remove the device and re-add, and I don't think configuration persists between remove/installs.